app/assets/stylesheets/member-facing/button.scss

```diff
@@ -4,10 +4,9 @@
   height: 60px;
   border: none;
   color: white;
-  text-transform: uppercase;
```

Comment:
@saraf22 Check for other templates as well

Reply:
changed